src/test/java/com/okta/tools/helpers/ProfileHelperTest.java

```diff
@@ -35,7 +36,7 @@
     @BeforeEach
     void setUp() {
         credentialsHelper = mock(CredentialsHelper.class);
-        environment = new OktaAwsCliEnvironment();
+        environment = new OktaAwsCliEnvironment(false, null, null, null, null, null, null, null, 0, fakeAwsRegion, false);
```

Comment:
For what it's worth, I couldn't find any other way to get this test to pass... if you have suggestions I'm open, but I think this is just due to how the initial constructor was designed... I don't really want to have to redesign it for the sake of this MR.